packages/http-client-csharp/generator/TestProjects/Plugin/Unbranded-TypeSpec/src/Generated/Models/AnonymousBodyRequest.Serialization.cs

```diff
@@ -0,0 +1,360 @@
+// <auto-generated/>
+
+#nullable disable
```

Comment:
why are there so many diffs if all this does is drop helper classes that aren't used?

Reply:
I think the plugin project was not up to date as we don't have it as part of the CI checks.